### ISSUE TITLE
Logging docs contain wrong import

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5083]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)| @hynek reported|
|-|-|
|Trac ID|trac#5083|
|Type|defect|
|Created|2011-05-06 08:27:49Z|

The example on http://twistedmatrix.com/documents/current/core/howto/logging.html for DailyLogFile has an invalid import:

```
from twisted.python.logging import DailyLogFile
```

It should be:

```
from twisted.python.logfile import DailyLogFile
```

Attachments:

* [5083-00-fix-logging-import.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/28a/28a721317b51cc6bc677d65335d266c51cc6bff2/229e86acc109da168a136c2feb3f4a44b06901a0.patch) (571 bytes) - added by ivank on 2011-05-06 08:36:44Z - fix

<details><summary>Searchable metadata</summary>

```
trac-id__5083 5083
type__defect defect
reporter__hynek hynek
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation_easy documentation easy
time__1304670469000000 1304670469000000
changetime__1304723287000000 1304723287000000
version__None None
owner__ 
cc__oubiwann cc__thijs cc__ivank
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="jerub's avatar" src="https://avatars.githubusercontent.com/u/315461?s=50" width="50" height="50">](https://github.com/jerub)<a name="note_3"></a>|@jerub commented|
|-|-|

I appreciate the quick turnaround on this. I am willing to commit this patch to trunk if a news file is correctly create. This is a doc fix so if you could amend the patch to contain 5083.doc with a brief statement about what was fixed, that would be best.

I've tested the new code to make sure it's okay, it doesn't do an explicit import of 'log' but considering the context that it is in, I don't think it's important for it to do so.

Please throw in a non-empty news file with a .doc extension, and I will commit your change.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra commented|
|-|-|

Replying to [jerub](#note_3):
> Please throw in a non-empty news file with a .doc extension, and I will commit your change.

I think this can be merged with an empty .misc file, because it's not major enough (like a new addition to the docs) to be mentioned in the changelog.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set status to `closed`|
|-|-|

(In [31776]) Apply logging howto example fix to import DailyLogFile from the right module

Author: ivank
Reviewer: jerub, thijs, exarkun
Fixes: [#5083](https://github.com/twisted/twisted/issues/5083)

Change an incorrect import to a correct import in the logging howto.
